demo/sections/components/RunGraphDemo.vue

```diff
@@ -1,23 +1,39 @@
 <template>
   <p-layout-default class="run-graph-demo">
-    <RunGraph :config="config" :zoom="zoom" class="run-graph-demo__graph" />
-    <p-number-input v-model="zoom" />
+    <RunGraph :config="config" class="run-graph-demo__graph" />
+    <!-- <p-number-input v-model="zoom" /> -->
   </p-layout-default>
 </template>
 
 <!-- eslint-disable camelcase -->
 <script lang="ts" setup>
-  import { endOfHour, startOfHour } from 'date-fns'
-  import { ref } from 'vue'
+  import { randomId } from '@prefecthq/prefect-design'
+  import { endOfHour, startOfHour, startOfDay, endOfDay } from 'date-fns'
+  // import { ref } from 'vue'
   import RunGraph from '@/components/RunGraph.vue'
   import { RunGraphConfig, RunGraphData } from '@/models'
 
-  const zoom = ref(0)
+  // const zoom = ref(0)
   const now = new Date()
 
   const dummy: RunGraphData = {
-    start_time: startOfHour(now),
-    end_time: endOfHour(now),
+    start_time: startOfDay(now),
+    end_time: endOfDay(now),
+    root_node_ids: [],
+    nodes: new Map([
+      [
+        randomId(), {
+          start_time: startOfHour(now),
+          end_time: endOfHour(now),
+          kind: 'task-run',
+          state_name: 'Completed',
```

Comment:
I know this is placeholder data but it's probably worth calling out now that we're likely to need state type in this payload as well if we want to support custom states in any capacity in the future... easier to get now than later

Reply:
Hmm that's a valid point

Reply:
Will follow up about this.